marlowe-cli/tests/Spec/Analysis.hs

```diff
@@ -345,7 +346,7 @@ protocolTestnet =
     , protocolParamStakeAddressDeposit = C.Lovelace 2000000
     , protocolParamStakePoolDeposit = C.Lovelace 500000000
     , protocolParamMinPoolCost = C.Lovelace 340000000
-    , protocolParamPoolRetireMaxEpoch = C.EpochNo 18
+    , protocolParamPoolRetireMaxEpoch = CI.EpochInterval 18
```

Comment:
All of these parameters should be upgraded to Conway sometime before the hard fork.